tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -477,22 +477,50 @@ When('the user disables all the custom permissions of collaborator {string} for
     .disableAllCustomPermissions(collaborator)
 })
 
-Then('custom permission/permissions {string} should be set for user {string} for file/folder {string} on the webUI', function (permissions, user, resource) {
-  return client.page
-    .FilesPageElement
-    .filesList()
-    .closeSidebar(100)
-    .openSharingDialog(resource)
-    .assertPermissionIsDisplayed(user, permissions)
-})
+const assertSharePermissions = async function (currentSharePermissions, permissions = undefined) {
+  let expectedPermissionArray
+  if (permissions !== undefined) {
+    expectedPermissionArray = await client.page.FilesPageElement.sharingDialog().getArrayFromPermissionString(permissions)
+  }
+  const COLLABORATOR_PERMISSION_ARRAY = ['share', 'update', 'create', 'delete']
```

Comment:
set in the PageObject (https://github.com/owncloud/phoenix/pull/2618/files#diff-c0efb50a13bceb9cb8549aa05fc0d250R239) and here, please DRY. Maybe move to sharing halper

Reply:
moved to sharingHelper!